backends/imgui_impl_dx9.cpp

```diff
@@ -70,6 +75,435 @@ static ImGui_ImplDX9_Data* ImGui_ImplDX9_GetBackendData()
     return ImGui::GetCurrentContext() ? (ImGui_ImplDX9_Data*)ImGui::GetIO().BackendRendererUserData : NULL;
 }
 
+// Shader pipeline
+static bool ImGui_ImplDX9_Shader_CreatePipeline()
+{
```

Comment:
Suggesting the rename the few functions
From
`ImGui_ImplDX9_Shader_XXXX`
To
`ImGui_ImplDX9WithShaderPipeline_XXXX`

This is a little more explicit that this is a "variant".

Reply:
I finally named it `ImGui_ImplDX9WithShader_XXXX`, it is simpler.